Description: Fix spelling errors found by lintian
Author: David Steele <steele@debian.org>
Forwarded: no
Last-Update: 2020-08-07

--- a/src/favorites.cpp
+++ b/src/favorites.cpp
@@ -171,7 +171,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",path ) ;
+		_log_error( "Unknown error has occurred",path ) ;
 	}
 
 	return {} ;
@@ -298,7 +298,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",a ) ;
+		_log_error( "Unknown error has occurred",a ) ;
 	}
 
 	return error::FAILED_TO_CREATE_ENTRY ;
--- a/src/engines/custom.cpp
+++ b/src/engines/custom.cpp
@@ -120,7 +120,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",s ) ;
+		_log_error( "Unknown error has occurred",s ) ;
 	}
 
 	return {} ;
--- a/src/3rdParty/lxqt_wallet/backend/lxqtwallet.h
+++ b/src/3rdParty/lxqt_wallet/backend/lxqtwallet.h
@@ -289,7 +289,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}else{
 			f = argv[ 3 ] ;
@@ -315,7 +315,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}else{
 			f = argv[ 3 ] ;
@@ -353,7 +353,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}
 	}else if( stringsAreEqual( command,"delete" ) ){
@@ -377,7 +377,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}
 	}else if( stringsAreEqual( command,"change" ) ){
@@ -401,7 +401,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}
 	}else if( stringsAreEqual( command,"value" ) ){
@@ -429,7 +429,7 @@
 			if( r == lxqt_wallet_wrong_password ){
 				puts( "wrong password" ) ;
 			}else{
-				puts( "general error has occured" ) ;
+				puts( "general error has occurred" ) ;
 			}
 		}
 	}else if( stringsAreEqual( command,"version" ) ){
--- a/src/createbackendwindow.cpp
+++ b/src/createbackendwindow.cpp
@@ -186,7 +186,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",path ) ;
+		_log_error( "Unknown error has occurred",path ) ;
 	}
 
 	this->Hide() ;
--- a/src/engines/fscrypt.cpp
+++ b/src/engines/fscrypt.cpp
@@ -581,7 +581,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",m_configFilePath ) ;
+		_log_error( "Unknown error has occurred",m_configFilePath ) ;
 	}
 
 	return {} ;
@@ -608,7 +608,7 @@
 
 	}catch( ... ){
 
-		_log_error( "Unknown error has occured",m_configFilePath ) ;
+		_log_error( "Unknown error has occurred",m_configFilePath ) ;
 	}
 }
 
--- a/src/utility.cpp
+++ b/src/utility.cpp
@@ -1337,7 +1337,7 @@
 
 			return m ;
 		}else{
-			utility::debug() << "Failed to get a responce from ykchalresp" ;
+			utility::debug() << "Failed to get a response from ykchalresp" ;
 			utility::debug() << "StdOUt:" << s.std_out() ;
 			utility::debug() << "StdError:" << s.std_error() ;
 		}
